client/src/pages/JobDetails.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from 'react';
+import { fetchOneJob, Job } from '../lib';
+import Footer from '../components/Footer';
+import { useParams } from 'react-router-dom';
+import JobInfoStack from '../components/JobInfoStack';
+import ReactMarkdown from 'react-markdown';
+
+export default function JobDetails() {
+  const { jobId } = useParams();
+  const [job, setJob] = useState<Job>();
+  const [loading, setLoading] = useState<boolean>();
+
+  useEffect(() => {
+    async function loadOneJob(jobId: number) {
+      setLoading(true);
+      try {
+        const job = await fetchOneJob(jobId);
+        setJob(job);
+      } catch (err) {
+        console.error(err);
```

Comment:
At some point you will want to go through and fix your error reporting. Users never see the console, so they'll miss that there was an error (maybe their phone doesn't have an internet connection). You should figure out how to display the error to the user.

Reply:
I've got this on my list, I'll make sure to do it soon so I don't have a ton of spots to update it in.